mantidproject/mantid
# First time gui shrunk a bit to fit smaller laptop screens

First time GUI has shrunk a bit to fit scaled or smaller screens

**To test:**
1. Look at the first time setup GUI
1. Check it fits on a windows 200% scaled laptop screen



Fixes #20607

**Release Notes** 

*Does not need to be in the release notes.*


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
